fenix/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -62,4 +62,9 @@ object FeatureFlags {
      * and managing search shortcuts in the quick search menu.
      */
     val unifiedSearchSettings = Config.channel.isNightlyOrDebug
+
+    /**
+     * Enables printing from the share and primary menu.
+     */
+    val print = Config.channel.isNightlyOrDebug
```

Comment:
After we add the nimbus experiment we could remove this
As we could control in which variant we want the feature enabled via 
`nimbus.fml.yam` an [example](https://github.com/mozilla-mobile/firefox-android/blob/d6b6f3c63cae8203c989775d9d24833a6c56dcb7/fenix/app/nimbus.fml.yaml#L241-L261) :)

Reply:
That's neat! Thanks for the link to the example!